index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>themeWagon</title>
+		<!-- <link rel="stylesheet" href="../task-html-and-css-basics/reset.css" /> -->
+		<script
+			src="https://kit.fontawesome.com/02b621b881.js"
+			crossorigin="anonymous"
+		></script>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="../task-html-and-css-basics/style.css" />
+	</head>
+	<body>
+		<header class="header">
+			<nav class="header__nav wrapper">
+				<div class="header__nav--logo">
+					<h1>nova</h1>
+				</div>
+				<div class="header__nav--menu" id="#" <a href="#"></a>
+					<a href="#team">The Team</a>
+					<a href="#pricing">Pricing</a>
+					<a href="#features">Features</a>
+				</div>
+			</nav>
+			<div class="header__front wrapper">
+				<div class="header__front--text">
+					<h2>Beautiful Free Nova template</h2>
+					<p>
+						Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam
+						exercitationem repellendus beatae!
+					</p>
+					<button class="header__front--btn">Download FREE</button>
+				</div>
+			</div>
+			<div class="header__front wrapper">
+				<div class="header__front--img"></div>
+			</div>
+			<div class="header__front--arrow-down">
+				<a href="#features" id="#features"<i class="fa-solid fa-arrow-down"></i></a>
+			</div>
+		</header>
+
+		<main id="features" class="main">
+			<div class="main__features wrapper">
+				<h2>features</h2>
+			</div>
+			<div class="main__features--list wrapper">
+				<div class="main__features--list-one display-flex">
```

Comment:
Tutaj również można użyć ul > li bo to lista cech więc ta struktura pasuje. Każde `li` to osobna cecha.